velox/core/SimpleFunctionMetadata.h

```diff
@@ -669,12 +669,13 @@ class UDFHolder {
   // Check which flavor of the call() method is provided by the UDF object. UDFs
```

Comment:
nit: not only call() method is mentioned here, so we need to update this comment.